src/test/test-network-forgiving.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+TEST_NET_RETRIES=${TEST_NET_RETRIES:-3}
+
+n_tries=0
+test_net_okay=0
+
+while [ "${n_tries}" -lt "${TEST_NET_RETRIES}" ]; do
+    n_tries=$((n_tries + 1))
+    # shellcheck disable=SC2086
+    if "${top_srcdir:-.}/src/test/test-network.sh" ${TEST_NETWORK_FLAGS}; then
```

Comment:
Should we use the script directory instead of the current working directory?
```suggestion
    if "${top_srcdir:-`dirname -- "$0"`}/src/test/test-network.sh" ${TEST_NETWORK_FLAGS}; then
```

Reply:
Hm. The script directory would mean that the default was something like scripts/test/src/test/test-network.sh, which won't work.  Maybe the default should be the script directory, followed by /../.. ?

Reply:
Yes, you're right, we need to go back those two steps, then forward.